app/workers/gitter_notificator.rb

```diff
@@ -0,0 +1,58 @@
+require 'gitter'
+require 'travis'
+
+class GitterNotificator
+  include Sidekiq::Worker
+  sidekiq_options :queue => :miq_bot_glacial, :retry => false
+
+  include Sidetiq::Schedulable
+  recurrence { minutely(10) }
+
+  include SidekiqWorkerMixin
+
+  def perform
+    init
+  end
+
+  private
+
+  def init
+    @gitter_token = Settings.travis_monitor.gitter_token
+
+    Settings.travis_monitor.branches.each do |element|
+      check(element.repo, element.branch, element.gitter_room)
+    end
+  end
+
+  def check(repository, branch, room)
+    builds = latest_builds(repository, branch, 2)
+
+    if builds[0] == 'passed' && builds[1] == 'failed'
+      msg = ":sos: :warning: \"#{branch}\" in \"#{repository}\" is broken :bangbang: :boom:"
+    elsif builds[0] == 'failed' && builds[1] == 'passed'
+      msg = ":white_check_mark: Broken branch has been fixed :green_heart:"
+    else
+      return
+    end
+
+    gitter_send(room, msg)
+  end
+
+  def latest_builds(repository, branch, count)
```

Comment:
Do you really need the `count` attribute? I don't think this method needs to be so universal.